docs-2.0/3.ngql-guide/3.data-types/5.null.md

```diff
@@ -4,6 +4,34 @@ You can set the properties for vertices or edges to `NULL`. Also, you can set `N
 
 If not specified, the property is set to `NULL` by default.
 
+## Logical operations with NULL
+
+<!-- The logcial operations with NULL is the same as OpenCypher. -->
+
+Here is the true table for AND, OR, XOR, and NOT.
+
+| a | b | a AND b | a OR b | a XOR | NOT a |
+| - | - | - | - | - | - |
+| false|  null|  false | null | null | true |
+| false | true | false | true | true | true|
+| true | false | false | true | true | false|
+| true|  null | null | true | null | false|
+| true | true | true | true | false | false|
+| null | false | false | null | null | null|
+| null | null | null | null | null | null|
+| null | true | null | true | null | null|
+
+## Comparisons with NULL
+
+TODO(incompatible)
+<!-- The comparison operations with NULL is incompatible with OpenCypher. -->
+
+## Operations and expression with NULL
```

Comment:
What about the `IN` operator and `null`?

Reply:
Current (2021.2.1) behavior is different with openCypher. 
I leave a TODO(coding) in the doc.